Base/QTCore/qSlicerCoreApplication.cxx

```diff
@@ -447,6 +447,11 @@ void qSlicerCoreApplicationPrivate::init()
     qDebug() << "Successfully uninstalled extension" << extensionName;
     }
 
+  // Set the list of installed extensions in the scene so that we can warn the user
+  // if the scene is loaded with some of the extensions not present.
+  QString extensionList = model->installedExtensions().join(";");
+  scene->SetExtensions(extensionList.toStdString().c_str());
```

Comment:
Can we also support extension versioning?

Reply:
If we release a new Slicer core version in every 3-6 months then probably we can keep extensions compatible with the corresponding Slicer core and we don't need additional versioning.